packages/multi-test/src/wasm.rs

```diff
@@ -720,6 +724,43 @@ where
         let storage = ReadonlyPrefixedStorage::multilevel(storage, &[NAMESPACE_WASM, &namespace]);
         Box::new(storage)
     }
+
+    fn verify_attributes(attributes: &[Attribute]) -> Result<(), String> {
+        for attr in attributes {
+            if attr.key.is_empty() {
```

Comment:
Note in the real wasmd implementation, we trim off whitespace from the ends before checking empty, so "  " is also invalid.

The relevant Go code changes are are https://github.com/CosmWasm/wasmd/pull/562/files maybe you can use them as reference

Reply:
Missed it, corrected.